doc/ex/NewCoordSys.rb

```diff
@@ -1,30 +1,30 @@
 require 'rvg/rvg'
 
 rvg = Magick::RVG.new(400, 150) do |canvas|
-    canvas.background_fill = 'white'
-    canvas.desc = 'Example NewCoordSys - New user coordinate system'
-    canvas.g.styles(:fill=>'none', :stroke=>'black', :stroke_width=>3) do |grp|
-        # Draw the axes of the original coordinate system
-        grp.line(0, 1.5, 400, 1.5)
-        grp.line(1.5, 0, 1.5, 150)
-    end
+  canvas.background_fill = 'white'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.